section4/exercises/burrito.rb

```diff
@@ -5,15 +5,31 @@
 # 3. change_protein
 
 class Burrito
+  
   attr_reader :protein, :base, :toppings
+  
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping()
+  end
+
+  def remove_topping()
+  end
+
+  def change_protein()
+  end
+
 end
 
 dinner = Burrito.new("Beans", "Rice", ["cheese", "salsa", "guacamole"])
 p dinner.protein
 p dinner.base
 p dinner.toppings
+
```

Comment:
Given above example, these three method calls would each have 1 argument.

Example of how the `dinner` instance of Burrito's state changes as we interact with the object using its `instance methods`... 

```rb
dinner.add_topping("queso")

dinner.toppings
  # => ["cheese", "salsa", "guacamole", "queso"]

dinner.remove_topping("cheese")
  # => "cheese"

dinner.toppings
  # => ["salsa", "guacamole", "queso"]

dinner.remove_topping("non existent topping")
  # => nil

dinner.toppings
  # => ["cheese", "salsa", "guacamole", "queso"]

dinner.change_protein("Chicken")
 
dinner.protein
  # => "Chicken"
```

These methods each have return values and `dinner`'s state is changing even though we don't print to the screen in the below interaction.